core/mm/phys_mem.c

```diff
@@ -26,9 +26,9 @@ static tee_mm_pool_t *init_pool(paddr_t b, paddr_size_t sz, uint32_t flags)
 		panic("invalid phys mem");
 
 	if (flags & TEE_MM_POOL_NEX_MALLOC)
-		pool = nex_malloc(sizeof(*pool));
+		pool = nex_calloc(1, sizeof(*pool));
 	else
-		pool = malloc(sizeof(*pool));
+		pool = calloc(1, sizeof(*pool));
```

Comment:
While `calloc()` is a safe choice, I'd expect `tee_mm_init()` to be able to initialize the pool correctly regardless.
How about fixing `tee_mm_init()`?

Reply:
I was about to suggest the same thing ;)

Reply:
Ok, that would be more consistent. I'll update. Thanks.

Reply:
Updated. I've preserved the `Fixes` tag in the commit message. Tell me if you think it's not really applicable.